CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Bug fixes
 
+* Prevent passing Proc as order (sort_key) parameter [#5464][] by [@chumakoff][]
```

Comment:
I find this description a bit confusing. This PR is not preventing a block being passed anywhere, right? What's doing is:
* Fixing the string representation of the resolved `sort_key` when no explicit `sortable` attribute is passed.
* Fixing docs on how the `sort_key` is chosen (which actually ignores the block passed to `column`).

If this is correct, can we clarify it?

Reply:
Thanks for updating the docs as well, by the way! :heart:

Reply:
I am fine with your description. Should I add it to the changelog as it is (two items)? Do you think fixing docs is worth mentioning in the changelog?

Reply:
I have the same doubt, but I guess it doesn't hurt, so I'd add both, yeah.

Reply:
Let's then formulate the second change like this:
* Fixed docs on the column `sortable` attribute (which is actually doesn't have to be explicitly specified when a block is passed to column)